### ISSUE TITLE
FeatureSet: Get training data and validation/test data

### ISSUE THREAD

**Issue by brifordwylie** [SuperCowPowers/workbench#190]
Lets put some helper methods on FeatureSet that give the training, validation, and test data.

**Comment by brifordwylie** [comment]
FeatureSets have a column called 'training' that is currently 1 if that row is to be used when training a model, folks can query this column to get either a training set or a test set.
